### ISSUE TITLE
End-user: Conditional logic accessibility testing

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#4212]
## Scope
Test the end-user experience when using conditional logic:
- Navigating pages
- Show/hide functionality 
- Checking the review page for mistakes and correcting them
- Submitting a form

**not in scope**
Public servant facing screens building the form. 

### Devices
- Mobile
- Tablet
- Desktop

### Technology
- Browsers: Chrome, MS Edge, Firefox, Safari
- Assistive tech:
  - Screen readers
  - 200% magnification for desktop
  - Voice to text
 
### Language
- English
- French

**Comment by thiessenp-cds** [comment]
Looks great! A few questions?
-200% magnification is probably good enough? (https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html#techniques)
-Do we care about Safari enough to put test time into it?

**Comment by srtalbot** [comment]
Thanks for reviewing Pete! 

Updated to 200%

As for Safari, it's actually our third most used browser for end-users!

**Comment by connorscarolyns** [comment]
Fable tests have been submitted. The results should be back by October 11.

**Comment by Abi-Nada** [comment]
Next steps are to break findings into actionable cards - we had removed this from the scope of the previous sprint to focus on API Concept Testing. We'll move it into PB and pull over when we have capacity

**Comment by srtalbot** [comment]
First round of findings have cards created.

**Comment by srtalbot** [comment]
@Abi-Nada - can this be closed?

**Comment by Abi-Nada** [comment]
Yes - this Epic can be closed as the testing has been completed, cards created and actioned on
